nixos/doc/manual/contributing-to-this-manual.chapter.md

```diff
@@ -11,3 +11,27 @@ $ nix-build nixos/release.nix -A manual.x86_64-linux
 ```
 
 If the build succeeds, the manual will be in `./result/share/doc/nixos/index.html`.
+
+**Contributing to the man pages**
+
+The manpages are written in [DocBook] which is XML.
+
+To see what your edits look like:
+
+```ShellSession
+$ cd /path/to/nixpkgs
+$ nix-build nixos/release.nix -A manpages.x86_64-linux
+```
```

Comment:
````suggestion
```
nix-build nixos/release.nix -A manpages.x86_64-linux
```
````

Reply:
This doesn't make it clear from where to execute the command. Someone coming from the `README.md` in the same folder will not know that they have to `cd` to the repo root in order to execute the command.

Additionally, this diverges from the format of the commands higher up in the file.

https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/contributing-to-this-manual.chapter.md?plain=1

Reply:
Fair, let's keep the format unified.